flowbased-computation/flowbased-computation-impl/src/main/java/com/farao_community/farao/flowbased_computation/impl/FlowbasedComputationImpl.java

```diff
@@ -67,6 +69,33 @@ public String getVersion() {
         SystematicSensitivityResult result = systematicSensitivityInterface.run(network);
         FlowbasedComputationResult flowBasedComputationResult = new FlowbasedComputationResultImpl(FlowbasedComputationResult.Status.SUCCESS, buildFlowbasedDomain(crac, glsk, result));
 
+        AtomicReference<String> curativeState = new AtomicReference<>("");
```

Comment:
Avoid usage of Atomic reference by replacing Stream.forEach() method by a standard for-each loop:
```java
for (Instant instant : crac.getInstants()) {
    if (instant.getSeconds() > seconds) {
        seconds = (double) instant.getSeconds();
        curativeState = instant.getId();
    }
}
```

Do the same kind of modification for the loop below

Reply:
Done, and I moved this to an independent method too.